pkg/apis/core/v4beta1/requirements.go

```diff
@@ -230,6 +230,8 @@ type IngressConfig struct {
 	CloudDNSSecretName string `json:"cloud_dns_secret_name,omitempty"`
 	// Domain to expose ingress endpoints
 	Domain string `json:"domain"`
+	// Extra domains to expose alternate endpoints with custom ingress
+	ExtraDomains []string `json:"extraDomains,omitempty"`
```

Comment:
Looks great, only one thought, I wonder if this should be a pointer so that extraDomains wouldn’t always appear in the jx-requirements.yaml, it would be more of an optional config like the TLSConfig?  
i.e
```
ExtraDomains *[]string `json:"extraDomains,omitempty"`
```

Reply:
Makes sense to me, I'll do it now